docs/src/stream.rst

```diff
@@ -224,4 +224,20 @@ API
 
     .. versionchanged:: 1.4.0 UNIX implementation added.
 
+.. c:function:: int uv_enable_largesocket(uv_stream_t* handle)
```

Comment:
A couple of things:
- Why a `uv_stream_t` instead of a `uv_tcp_t` as this addition is for TCP only?
- What about renaming it to something like `uv_tcp_enable_largesocket`?